src/import-tool.cc

```diff
@@ -350,17 +356,25 @@ string get_file_modification_time(string file_path) {
     return format_utc_epoc_count_as_local_time(last_modification_time);
 }
 
+// this function assumes importer arg is valid
+int request_importer_start(string importer, string start_trigger_filename, string kill_trigger_filename, string killing_filename) {
+    if (file_exists(killing_filename)) {
+        cerr << "Cannot trigger importer start now because running importer is now being killed (wait a minute or two and try again)" << endl;
+        return 1;
+    }
+    if (file_exists(start_trigger_filename)) {
+        cerr << "Cannot trigger importer start now because importer kill is currently triggered (wait a minute or two and try again)" << endl;
+        return 1;
+    }
+    return create_trigger_file(START_TRIAGE_IMPORT_TRIGGER_FILENAME);
```

Comment:
this is a bug .. the constant START_TRIAGE_IMPORT_TRIGGER_FILENAME should be start_trigger_filename